addons/account/report/inherited_layouts.xml

```diff
@@ -6,7 +6,7 @@
     <xpath expr="//ul[@name='financial_infos']" position="inside">
         <t t-if="not company.custom_footer">
             <t t-set="is_first" t-value="True"/>
-            <t t-foreach="company.bank_journal_ids" t-as="journal">
+            <t t-foreach="company.bank_journal_ids.sudo()" t-as="journal">
```

Comment:
IMHO, a record rule is more secure, in some places such information is security sensible to force sudo vía template, but... well, you should wait for this kind of extreme merges at least 1 hour make commit-automerge exposing with sudo a variable is not ok dude.... (just thinking aloud) even if you are right.